src/styles/blocks/main/_catalog.scss

```diff
@@ -0,0 +1,33 @@
+.catalog {
+  box-sizing: content-box;
```

Comment:
use `border-box` across the whole page (it's strange if there is `content-box` and somewhere else `border-box`)

it's better to set for each element on the page (use `*` selector)